tests/common/devices/sonic.py

```diff
@@ -762,6 +762,9 @@ def get_pmon_daemon_states(self):
                 else:
                     logging.debug("Daemon %s is disabled" % key)
                     exemptions.append(key)
+
+            if self.is_multi_asic and self.sonic_release in ['201911']:
```

Comment:
Should we add platform_api_server as a common exemption? without checking for release/asic.
This is not really a pmon daemon, it is a sever spawned to test the platform api's.

Reply:
I don't know about single ASIC, @sujinmkang  can you comment on this question.